src/main/resources/application.yml

```diff
@@ -161,8 +161,9 @@ gcp:
   receiptSubscription: ras-rm-receipt
   caseNotificationTopic: "test_topic"
   caseNotificationSubscription: "test_subscription"
-  printFileTopic: print-file-sinhaa
-  notifyTopic: ras-rm-notify-sinhaa
+  collectionExerciseEventStatusTopic: "test_event_status_topic"
+  printFileTopic: "test_notify_letter_topic"
+  notifyTopic: "test_notify_email_topic"
   bucket:
     name: ras-rm-print-file-sinhaa
     prefix: ""
```

Comment:
add new line

Reply:
sorry not sure I get it. what do we mean by add a new line?